identifiers/country-pt/regions.csv

```diff
@@ -0,0 +1,24 @@
+id,name
+ocd-division/country:pt,Portugal
+ocd-division/country:pt/region:av,Aveiro
```

Comment:
Please use "state" instead of "region" in the canonical OCD-IDs, as "state" is used to represent the first-level administrative area inside of countries. "/district:" can be used in the aliases.csv file.

https://en.wikipedia.org/wiki/Districts_of_Portugal

> The Districts of Portugal (Portuguese: Distritos de Portugal), are the most important first-level administrative subdivisions of mainland Portugal.

Reply:
We need to change the feed in this case, don't we ?

Reply:
We use region in the feed